ChangeLog

```diff
@@ -46,6 +46,13 @@ Changes
      mbedtls_ssl_get_input_max_frag_len() to ensure that a sufficient input
      buffer is allocated by the server (if MBEDTLS_SSL_VARIABLE_BUFFER_LENGTH
      is defined), regardless of what MFL was configured for it.
+   * Add the command line parameter key_pwd to the ssl_client2 and ssl_server2
+     example applications which allows to provide a password for the key file
+     specified through the existing key_file argument. This allows the use of
+     these applications with password-protected key files. Analogously but for
+     ssl_server2 only, add the command line parameter key_pwd2 which allows to
+     set a password for the key file provided through the existing key_file2
+     argument.
```

Comment:
We've moved to using ChangeLog entries instead of directly editing the ChangeLog to avoid conflicts. More guidance here: https://github.com/ARMmbed/mbedtls/blob/development/ChangeLog.d/00README.md